project/plugins.sbt

```diff
@@ -5,3 +5,7 @@ addSbtPlugin("com.github.gseitz" % "sbt-release" % "1.0.3")
 addSbtPlugin("org.scoverage" % "sbt-scoverage" % "1.3.5")
 
 addSbtPlugin("org.scoverage" % "sbt-coveralls" % "1.1.0")
+
+addSbtPlugin("com.timushev.sbt" % "sbt-updates" % "0.3.0")
+
+addSbtPlugin("io.get-coursier" % "sbt-coursier" % "1.0.0-M15-1")
```

Comment:
This is very interesting, but again, doesn't seem lenthall specific. Also the doc stating that the plugin doesn't work with sbt 1.0 makes me feel like we'd need to keep this plugin up to date soon, while instead it could easily live outside the project as a global plugin also.